liwords-ui/public/index.html

```diff
@@ -45,7 +45,7 @@
       rel="stylesheet"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;500;700;900&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;500;700;800;900&display=swap"
       rel="stylesheet"
```

Comment:
- let's close the link tag -- `/>` in line 50
- i think we need a few more weights; we see 300, 600, 1000, 200 in various of the scss files (or should we edit those?)

```shell
~/git/liwords$ git grep -Pin '\bfont-weight:(?!\s*(?:400|500|700|900|normal|bold|black|inherit)\b\s*;)'
liwords-ui/src/App.scss:117:      font-weight: 600;
liwords-ui/src/base.scss:183:  font-weight: 800;
liwords-ui/src/base.scss:655:    font-weight: 800;
liwords-ui/src/base.scss:768:      font-weight: 600;
liwords-ui/src/base.scss:775:        font-weight: 1000;
liwords-ui/src/base.scss:786:        font-weight: 600;
liwords-ui/src/chat/chat.scss:198:        font-weight: 800;
liwords-ui/src/chat/chat.scss:228:      font-weight: 200;
liwords-ui/src/chat/chat.scss:313:        font-weight: 800;
liwords-ui/src/chat/chat.scss:352:        font-weight: 800;
liwords-ui/src/gameroom/scss/gameroom.scss:423:    font-weight: 800;
liwords-ui/src/gameroom/scss/gameroom.scss:869:  font-weight: 600;
liwords-ui/src/gameroom/scss/gameroom.scss:1133:    font-weight: 600;
liwords-ui/src/gameroom/scss/gameroom.scss:1201:      font-weight: 800;
liwords-ui/src/gameroom/scss/gameroom.scss:1643:      font-weight: 600;
liwords-ui/src/lobby/accountForms.scss:26:      font-weight: 800;
liwords-ui/src/lobby/lobby.scss:81:        font-weight: 300;
liwords-ui/src/settings/settings.scss:120:      font-weight: 800;
```

Reply:
I will edit them out, if they're for Muli (some of them aren't).